tools/functional-tester/etcd-tester/checks.go

```diff
@@ -46,87 +45,32 @@ func newHashChecker(hrg hashAndRevGetter) Checker { return &hashChecker{hrg} }
 const leaseCheckerTimeout = 10 * time.Second
 
 func (hc *hashChecker) checkRevAndHashes() (err error) {
-	// retries in case of transient failure or etcd nodes have not stablized yet.
 	var (
-		revsStable   bool
-		hashesStable bool
+		revs   map[string]int64
+		hashes map[string]int64
 	)
+
+	// retries in case of transient failure or etcd cluster has not stablized yet.
 	for i := 0; i < retries; i++ {
-		revsStable, err = hc.areRevisonsStable()
-		if err != nil || !revsStable {
-			continue
-		}
-		hashesStable, err = hc.areHashesStable()
-		if err != nil || !hashesStable {
-			continue
+		revs, hashes, err = hc.hrg.getRevisionHash()
+		if err != nil {
+			plog.Warningf("retry %i. failed to retrieve revison and hash (%v)", i, err)
+		} else {
+			_, sameRev := getSameValue(revs)
+			_, sameHashes := getSameValue(hashes)
+			if sameRev && sameHashes {
```

Comment:
change getSameValue to only return bool to get `if sameValues(revs) && sameValues(hashes)`? nothing seems to be using the value